src/test/java/com/commercetools/testUtil/ctpUtil/CleanupTableUtil.java

```diff
@@ -46,10 +55,12 @@ public static int cleanupPaymentTable(SphereClient sphereClient) {
      * @param <EntityType>   type of items to read/delete
      * @return number of read/deleted items
      */
-    public static <EntityType> int cleanupTable(
+    private static <EntityType> int cleanupTable(
             @Nonnull final SphereClient client,
             @Nonnull final Supplier<SphereRequest<PagedQueryResult<EntityType>>> querySupplier,
-            @Nonnull final Function<EntityType, SphereRequest<EntityType>> deleteFunction) {
+            @Nonnull final Function<EntityType, SphereRequest<EntityType>> deleteFunction,
+            @Nonnull String resourceName) {
+        logger.debug("Cleanup " + resourceName + " table");
```

Comment:
`logger.debug("Cleanup {} table", resourceName);`

see https://www.slf4j.org/manual.html#typical_usage
and  https://www.slf4j.org/faq.html#logging_performance